mongodb/configure.sls

```diff
@@ -121,3 +121,12 @@ configure_keyfile_and_replicaset:
     - watch:
         - file: configure_keyfile_and_replicaset
 {% endif %}
+
+# Make sure that the log file can be read by group 'adm' so that logging agents
+# can read it.
+ensure_ownership_and_perms_of_logfile:
+  file.managed:
+    - name: /var/log/mongodb/mongodb.log
+    - user: mongodb
+    - group: adm
```

Comment:
Where/how does this group get created?

Reply:
`adm` is an age-old Unix group that's traditionally used to own log files and statistics-related files. I think that utilities like `sar` and `syslog` use it... do a `ls -l` of `/var/log` ...

The Vector people are observing the tradition by making Vector a member of the `adm` group by default.